src/components/Header/Header.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import React, { useEffect, useRef } from 'react';
+import { Todo } from '../../types/Todo';
+import * as todosService from '../../api/todos';
+
+type Props = {
+  todos: Todo[];
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  todoTitle: string;
+  setTodoTitle: (newTitle: string) => void;
+  setErrorMessage: (errorMessage: string) => void;
+  loading: boolean;
+  setLoading: (loadingStatus: boolean) => void;
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  setTodos,
+  todoTitle,
+  setTodoTitle,
+  setErrorMessage,
+  loading,
+  setLoading,
+}) => {
+  const titleField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (titleField.current) {
+      titleField.current.focus();
```

Comment:
it doesn't work (and tests also fails because of it) - try to debug and find the reason. Ask for help in the chat if you need